_events/2023-0718-2-9-release-meetings.markdown

```diff
@@ -1,11 +1,10 @@
 ---
 
-
-eventdate: 2023-07-19 15:00:00 -0700
-title: OpenSearch 2.9.0 Release Meetings - 2023-07-18
+eventdate: 2023-07-24 11:30:00 -0700
+title: OpenSearch 2.9.0 Release Meetings - 2023-07-24
 online: true
 signup:
-  url: https://us02web.zoom.us/j/83030994833
+  url: https://www.meetup.com/opensearch/events/tvcvftyfclbcc/
```

Comment:
:warning:  The new link is the one for "Development Backlog & Triage Meeting - Security".  Typo?